Makefile

```diff
@@ -620,7 +621,7 @@ run-android-upload-archives: platform/android/configuration.gradle
 # Runs Android UI tests on all connected devices using Spoon
 .PHONY: run-android-ui-test-spoon
 run-android-ui-test-spoon: platform/android/configuration.gradle
-	cd platform/android && $(MBGL_ANDROID_GRADLE) -Pmapbox.abis="$(MBGL_ANDROID_ACTIVE_ARCHS)" spoon
+	cd platform/android && $(MBGL_ANDROID_GRADLE) -Pmapbox.abis=all spoon
```

Comment:
for doing [spoon](https://github.com/square/spoon) runs, having default set to all seems most logical.